components/organisms/Contributors/contributors.tsx

```diff
@@ -31,17 +31,18 @@ import ContributorTable from "../ContributorsTable/contributors-table";
 interface ContributorProps {
   repositories?: number[];
   title?: string;
+  defaultLayout?: ToggleValue;
 }
 
-const Contributors = ({ repositories, title }: ContributorProps): JSX.Element => {
+const Contributors = ({ repositories, title, defaultLayout }: ContributorProps): JSX.Element => {
```

Comment:
nit: since `defaultLayout` is optional, set the default value here instead of via `??`.

```suggestion
const Contributors = ({ repositories, title, defaultLayout = "list" }: ContributorProps): JSX.Element => {
```